docs/core/compatibility/deployment/7.0/multilevel-lookup.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Breaking change: Multi-level lookup is disabled"
+description: Learn about the .NET 7 breaking change in deployment where multi-level lookup is disabled.
+ms.date: 04/08/2022
+---
+# Multi-level lookup is disabled
+
+On Windows, framework-dependent .NET applications no longer search for frameworks in multiple install locations.
+
+## Previous behavior
+
+In previous versions, a framework-dependent .NET application searched for frameworks in multiple install locations on Windows. The locations were:
+
+- When running the application through [dotnet](../../../tools/dotnet.md), subdirectories relative to the `dotnet` executable.
+- When running the application through its executable (`apphost`), the `DOTNET_ROOT` environment variable (if set).
+- The globally registered install location (if set) in **HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\<arch>\InstallLocation**.
+- The default install location of *%ProgramFiles%\dotnet* (or *%ProgramFiles(x86)%\dotnet* for 32-bit processes on 64-bit Windows).
+
+This multi-level lookup behavior was enabled by default but could be disabled by setting the environment variable `DOTNET_MULTILEVEL_LOOKUP=0`.
+
+## New behavior
+
+Applications that target .NET 7 or a later version only look for frameworks in one location, which is the first location where a .NET installation is found. When running an application through [dotnet](../../../tools/dotnet.md), frameworks are only searched for in subdirectories relative to the `dotnet` executable. When running an application through its executable (`apphost`), frameworks are only searched for in the first of the locations listed in the [previous behavior section](#previous-behavior) where .NET is found.
```

Comment:
>only searched for in the first of the locations listed in the [previous behavior section](#previous-behavior) where .NET is found.

The first location is "When running the application through [dotnet] ... subdirectories relative to the `dotnet` executable."